numba/tests/test_interpreter.py

```diff
@@ -0,0 +1,641 @@
+                a='int64',
+                b='int64',
+                c='int64',
+                d='int64',
+                e='int64',
+                f='int64',
+                g='int64',
+                h='int64',
+                i='int64',
+                j='int64',
+                k='int64',
+                l='int64',
+                m='int64',
+                n='int64',
+                o='int64',
+                p='int64',
+            ):
+                (
+                    a,
+                    b,
+                    c,
+                    d,
+                    e,
+                    f,
+                    g,
+                    h,
+                    i,
+                    j,
+                    k,
+                    l,
+                    m,
+                    n,
+                    o,
+                    p
+                ) = py_func()
+            return (
+                a
+                + b
+                + c
+                + d
+                + e
+                + f
+                + g
+                + h
+                + i
+                + j
+                + k
+                + l
+                + m
+                + n
+                + o
+                + p
+            )
+
+        a = sum(list(py_func()))
+        b = objmode_func()
+        self.assertEqual(a, b)
+
+    @skip_unless_py10
+    def test_large_args_inline_controlflow(self):
+        """
+        Tests generating large args when one of the inputs
+        has inlined controlflow.
+        """
+        def inline_func(flag):
+            return sum_jit_func(
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1 if flag else 2,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                1,
+                arg41=1,
+            )
+
+        with self.assertRaises(UnsupportedError) as raises:
+            njit()(inline_func)(False)
+        self.assertIn(
+            'You can resolve this issue by moving the control flow out',
+            str(raises.exception)
+        )
+        # Uncomment sections when inlined control flow is
+        # actually supported.
+        # py_func = inline_func
+        # cfunc = njit()(inline_func)
+        # a = py_func(False)
+        # b = cfunc(False)
+        # self.assertEqual(a, b)
+
+    @skip_unless_py10
+    def test_large_args_uninlined_controlflow(self):
```

Comment:
```suggestion
    def test_large_args_noninlined_controlflow(self):
```
(suggest same change throughout).

Reply:
Done.